script/ci

```diff
@@ -7,4 +7,4 @@ set -e
     ./script/ci
 )
 
-./sbt --no-conf '; project hq; clean; compile; riffRaffAddManifest; riffRaffUpload'
\ No newline at end of file
+sbt "project hq" "; clean; compile; riffRaffUpload"
```

Comment:
I wonder if these lines are important?

https://github.com/guardian/security-hq/blob/main/sbt#L27-L36

When I originally worked on SHQ the *build* didn't require any configuration, they were just for running it. That may not be the case anymore, if someone has added these settings? I guess we'll need to look into it. We may also need to double check that the GHA migration has handled this, assuming it is important.

Reply:
I also don't think the semicolons are required, although I can see it wasn't you that did it this way.

Clearer (or at least more conventional) might be:

`sbt "project hq" clean compile riffRaffUpload`

Reply:
IIUC those lines are efforts to help `sbt-riffraff-artifact`, specifically providing information for the [`build.json` file](https://github.com/guardian/sbt-riffraff-artifact/blob/548fabb329c292e2eb1f0351836edd9219d48e5e/src/main/scala/com/gu/riffraff/artifact/BuildInfo.scala#L11-L16).

`sbt-riffraff-artifact` is actually [already](https://github.com/guardian/sbt-riffraff-artifact/blob/548fabb329c292e2eb1f0351836edd9219d48e5e/src/main/scala/com/gu/riffraff/artifact/BuildInfo.scala#L100-L139) reading the TC property file, so we no longer need to 🎉 .

The version of `sbt-riffraff-artifact` we're using is GHA aware too. That is, those lines can be removed entirely.

Reply:
I propose we remove the redundant TC lines in a separate PR as it's a different concern.